submissions/Mifaresss/js_dom/script.js

```diff
@@ -0,0 +1,126 @@
+		<p class="title-description__text">Правда всё не так гладко, как кажется на первый взгляд. В приюте «Грэйс Филд», среди таких же сирот, живут 11-летние Норман, Эмма и Рэй. Они дружат всю жизнь и отличаются любознательностью, из-за которой часто попадают в передряги. Все подопечные приюта мечтают о приемной семье, и ждут, когда их заберут новые родители, но пока ребята беззаботно развлекаются, играют в игры. Мама приюта – смотрительница Изабелла, тщательно заботится о воспитанниках. Она просит детей соблюдать лишь одно условие – дети не могут выходить из детского дома. Каждый ребенок в приюте должен выполнять этот закон.</p>`
+	},
+	{
+		id: 9,
+		header: 'Черный клевер',
+		img: './images/black-clever.jpg',
+		subHeader: 'Краткое описание',
+		description: `Их свела общая судьба. Аста и Юно – двое сирот, мечтающих найти своё место под солнцем и когда-нибудь прославиться как великие маги. Однако общая судьба - не общий дар, и, как это часто бывает, пока один достигает определённых успехов, другой вынужден плестись позади, не проявляя практически никакого таланта. Но друг, как известно, познаётся в беде. Когда Юно, получившему могущественный гримуар четырёхлистного клевера, угрожает опасность, Аста раскрывает весь свой потенциал и получает гримуар, на котором красуется клевер с пятью лепестками, прозванный «Чёрным».`
+	},
+];
+
+const cardsList = document.querySelector('.main__content');
+
+function createCards(id, header, img, subHeader, description) {
+	const cardsHTML = `
+		<li class="main__content-item ${id}">
+			<h2 class="sub-title">${header}</h2>
+			<img src="${img}" alt="${header}" class="main__img">
+			<h3 class="title-description">${subHeader}</h3>
+			<p class="title-description__text">${description}</p>
+		</li>
+		`
+	cardsList.innerHTML = cardsHTML;
+}
+
+function showCard(event) {
+	if (event.target.tagName !== 'BUTTON') return false;
+	const filterClass = event.target.dataset.f;
+	const { id, header, img, subHeader, description } = cards.find(card => card.id == filterClass);
+	createCards(id, header, img, subHeader, description);
+	const homeScreen = document.querySelector('.home-screen');
+	homeScreen.classList.add('display-none');
+}
+function hidingBurger(event) {
+	if (event.target.tagName !== 'BUTTON') return false;
+	if (iconMenu.classList.contains('_active')) {
+		iconMenu.classList.remove('_active');
+		buttons.classList.remove('_active');
+		document.body.classList.remove('_lock');

```

Comment:
You twice duplicated this part of the code. Create a function for it.

Reply:
I didn't understand here. In the first case, I work with clicks on the navigation buttons (so that when the burger menu is active, when the button is clicked, it closes). In the second case, I work with the menu button itself. Should I just name the functions more correctly or create another one?

Reply:
Made code refactoring